backend/lib/endpoints/schema/organizations.js

```diff
@@ -1,15 +1,39 @@
 const S = require("fluent-schema");
 const { strictSchema } = require("./utils");
+const { locationSchema } = require("./location");
+
+const organization = {
+  about: S.string(),
+  email: S.string(),
```

Comment:
do we have email verification for an organization?
Or should we only use an email from a user?
Currently the email verification for user accounts is completely handled by Auth0 and we should be careful with saving any email address to our own database, I would prefer to let Auth0 handle it for now.